artifacts/rhdh-plugins-reference/quickstart/quickstart-plugin-admin.adoc

```diff
@@ -0,0 +1,55 @@
+[id="installing-configuring-quickstart-plugin"]
+= Installing and configuring the Quickstart plugin
+
+The Quickstart plugin provides a guided onboarding experience for new users of {product}. It displays a customizable drawer interface with interactive quickstart steps that help users get familiar with the platform.
```

Comment:
Currently if permissions are enabled quickstart is only accesible to admins and if permissions are not enabled then anyone can access it.